docs/_src/usage/usage/retriever.md

```diff
@@ -122,6 +122,14 @@ Indexing using DPR is comparatively expensive in terms of required computation s
 The embeddings that are created in this step can be stored in FAISS, a database optimized for vector similarity.
 DPR can also work with the ElasticsearchDocumentStore or the InMemoryDocumentStore.
 
+<div class="recommendation">
+
+**Tip**
+
+When using DPR, it is recommended that you use the dot product similarity function since that is how it is trained.
```

Comment:
Great idea! I like these recommendation boxes. How about stating there directly how to configure this in the docstore? It's great if the user is aware of using a certain similarity function, but if he/she doesn't know **how** to change that it's useless. Similar for the logging further below